xcode/Subconscious/Shared/Parsers/SubtextAttributedStringRenderer.swift

```diff
@@ -113,17 +141,127 @@ struct SubtextAttributedStringRenderer {
     /// You can use this to produce strings which can be rendered in
     /// `Text` blocks.
     func render(_ string: String) -> AttributedString {
-        let nsAttributedString = NSMutableAttributedString(
-            string: string
-        )
-        _ = renderAttributesOf(nsAttributedString)
-        return AttributedString(nsAttributedString)
+        let dom = Subtext(markup: string)
```

Comment:
Unfortunately, converting from `NSAttributedString` to `AttributedString` proved to be lossy. We lost the links in particular.

I ended up-re-implementing the renderer in terms of the new API.